generic/tests/dd_test.py

```diff
@@ -83,6 +90,7 @@ def _get_file(filename, select, test=test):
 
     dev_partitioned = []
     for arg in ['dd_if', 'dd_of']:
+        partitions = []
```

Comment:
`partitions` is used only for the `if` block, no need to put it here.

Reply:
thanks, have updated.